lib/puppet/provider/service/systemd.rb

```diff
@@ -46,11 +67,22 @@ def status
   end
 
   def enable
+    output = systemctl("unmask", @resource[:name])
```

Comment:
We discussed whether `enable => true` should unmask and enable a service; this will hide the fact that a service was masked and may make it easier to unintentionally unmask a service. Is this acceptable?